app/mailers/notifier.rb

```diff
@@ -12,7 +12,6 @@ class Notifier < ActionMailer::Base
   # Who created the account.  How to update.
   def new_user(args)
```

Comment:
Not a big deal, but should we change the arguments to now take the `user` keyword and make it required?